src/containers/TriggerTemplates/TriggerTemplates.js

```diff
@@ -25,14 +25,17 @@ import { FormattedDate, Table } from '@tektoncd/dashboard-components';
 import { Link as CarbonLink } from 'carbon-components-react';
 
 import { ListPageLayout } from '..';
-import { useTriggerTemplates } from '../../api';
-import { getSelectedNamespace, isWebSocketConnected } from '../../reducers';
+import { useSelectedNamespace, useTriggerTemplates } from '../../api';
+import { isWebSocketConnected } from '../../reducers';
 
 function TriggerTemplates(props) {
-  const { filters, intl, selectedNamespace, webSocketConnected } = props;
+  const { filters, intl, match, webSocketConnected } = props;
 
   useTitleSync({ page: 'TriggerTemplates' });
 
+  const { selectedNamespace: defaultNamespace } = useSelectedNamespace();
+  const selectedNamespace = match.params.namespace || defaultNamespace;
```

Comment:
Nit. Think I prefer the style you used here: https://github.com/tektoncd/dashboard/pull/2141/files?diff=split&w=1#diff-d4255f636c1984d552c8f721d92957aa4156f5469cc8fcd47303a72faa5aaf42R36. Same effect

`const { namespace = selectedNamespace } = props.match.params;`

Reply:
Though in this case, due to using `selectedNamespace` instead of `namespace` maybe its not viable

Reply:
Ah you have it handled here for the rename situation: https://github.com/tektoncd/dashboard/pull/2141/files?diff=split&w=1#diff-f7fea6067540b83411ae8d716f05c9ed4cb9c32eadf17b61526054034e772504R37. Maybe the use of `||` is more readable/preferable in this case?

Reply:
I'll try it out and see how it looks (might be a bit confusing with the renaming and defaulting all happening at once), and if it's no good we can stick with `||`. We could probably do with improving some of the naming anyway so let's give it a go.